website/docs/api/admin/segments.mdx

```diff
@@ -280,44 +280,83 @@ Use a JSON object with the following properties to update the list of applied se
 
 ## Types
 
-### `ISegment`: segment interface {#isegment}
+The segments API exposes the following types:
```

Comment:
Maybe something like this, since it's not quite types anymore:

```
## Response JSON 

Here are some example JSON responses from the segments API:
```

Reply:
Agreed! Done 😄